Task3/src/main/java/model/HouseManager.java

```diff
@@ -0,0 +1,78 @@
+package model;
+
+import model.comparator.DistanceToKindergartenComparator;
+import model.comparator.DistanceToPlaygroundComparator;
+import model.comparator.DistanceToSchoolComparator;
+import model.comparator.PriceComparator;
+
+import java.util.*;
+
+
+public class HouseManager  {
```

Comment:
You should add methods that you use in pressButton1...5 to Model and then HouseManager should implement Model